grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -78,6 +79,23 @@ class GrabbitConfiguration {
         )
     }
 
+    private
+    static Collection<String> nonEmptyCollection(Collection<String> excludePaths, ConfigurationException.Builder errorBuilder) {
+        String prePattern = "^(/|\\./|\\|\\\\).*\$";
+        String postPattern = ".*(/|\\./|\\|\\\\)\$";
```

Comment:
BTW, if you didn't know, if you Alt-Enter inside the regex you can select "Check RegExp" and it will bring up a dialog to help test/build your regular expression. http://regexr.com/ and http://regexpal.com/ can be used for that too.